save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestSuitesService.kt

```diff
@@ -91,7 +91,7 @@ class TestSuitesService(
                 .withMatcher("name", ExampleMatcher.GenericPropertyMatchers.contains().ignoreCase())
                 .withMatcher("language", ExampleMatcher.GenericPropertyMatchers.contains().ignoreCase())
                 .withMatcher("tags", ExampleMatcher.GenericPropertyMatchers.contains().ignoreCase())
-                .withIgnorePaths("description", "source", "version", "dateAdded", "plugins")
+                .withIgnorePaths("description", "source", "version", "dateAdded", "plugins", "sourceSnapshot")
```

Comment:
`source` can be removed and please add a link/note about this runtime dependency to entity class

Reply:
Replaced `source` with `sourceSnapshot` as well as added reminder to `TestSuite`'s KDoc